designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/theme/Shadow.kt

```diff
@@ -56,12 +58,12 @@ internal fun Modifier.buttonShadow(
             val paint = Paint()
             val frameworkPaint = paint.asFrameworkPaint()
             val spreadPixel = if (hasIcon) Spacing.Spacing0.toPx() else Spacing.Spacing1_5.toPx()
-            val leftPixel = DEFAULT_PADDING
+            val leftPixel = 0f
```

Comment:
is there any Spacing value matching 0f?

Reply:
there is a Spacing value of 0.dp not without the dp conversion, as pixels are needed for this. I am going to fix all this with the Refactor Spacings issue once the current PRs have been merged